### DISCUSSION TITLE
محدود کردن اتصال همزمان کاربران

### DISCUSSION THREAD

**Discussion by smtk69** [hiddify/Hiddify-Manager#3166] [Ideas ایدهها]
لطفا قابلیت محدودکردن اتصال همزمان به یک کانکشن رو بزارید، که مثلا چندتا آی پی میتونن به یک کانکشن وصل بشن

**Comment by S13cko** [comment]
واقعا لازمه

**Comment by zZedix** [comment]
این امکان هرگز به پنل اضافه نخواهد شد دلایلش هم بارذکر شده
۱ - در حالت CDN و تانل تمامی ای پی ها به یک ای پی تبدیل میشن عملا اعمال محدودیت ممکن نیست
۲ - این موضوع منابع بیشتری رو در سرور درگیر میکنه و باعث میشه پروژه سنگین تر بشه
شما میتونید از همون محدودیت حجم استفاده کنید
